lib/df-detection.c

```diff
@@ -10,7 +10,8 @@
 #include <linux/sys.h>
 #include <linux/types.h>
 
-void add_address(const void *addr, size_t len, unsigned long caller)
+void add_address(const void *addr, size_t len, unsigned long caller,
```

Comment:
You also need to update the function prototype in include/linux/df-detection.h, without it the kernel doesn't compile.